core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -11111,7 +11111,7 @@ class KafkaApisTest extends Logging {
     kafkaApis.handle(requestChannelRequest, RequestLocal.noCaching)
 
     val expectedHeartbeatResponse = new ConsumerGroupHeartbeatResponseData()
-      .setErrorCode(Errors.UNSUPPORTED_VERSION.code)
+      .setErrorCode(Errors.UNKNOWN_SERVER_ERROR.code)
```

Comment:
this test case is used to verify the following check.
```java
  def isConsumerGroupProtocolEnabled(): Boolean = {
    groupCoordinator.isNewGroupCoordinator &&
      config.groupCoordinatorRebalanceProtocols.contains(Group.GroupType.CONSUMER) &&
      groupVersion().isConsumerRebalanceProtocolSupported
  }
```

That means we should create kafkaApis based on KRaft in this test case, rather than just changing the error code.